src/common/transformations/tests/op_conversions/fake_convert_decomposition_test.cpp

```diff
@@ -50,75 +60,58 @@ class FakeConvertDecompositionTest : public ov::test::TestsCommon,
-        } else {
-            const auto shift = std::make_shared<ov::op::v1::Subtract>(scale, input_shift);
-
-            const auto clamp = std::make_shared<ov::op::v0::Clamp>(shift, lower_bound, upper_bound);
-            const auto downconvert = std::make_shared<ov::op::v0::Convert>(clamp, dst_prec);
-            const auto upconvert = std::make_shared<ov::op::v0::Convert>(downconvert, data_prec);
-
-            const auto deshift = std::make_shared<ov::op::v1::Add>(upconvert, input_shift);
-            result = std::make_shared<ov::op::v1::Divide>(deshift, input_scale);
+protected:
+    void SetUp() override {
+        FakeConvertDecompositionTestBase::SetUp();
+        const auto [data_shape, scale_shape, shift_shape, data_prec, dst_prec, default_shift] = this->GetParam();
+        {
+            const auto data = std::make_shared<opset1::Parameter>(data_prec, PartialShape(data_shape));
+            const auto scale = std::make_shared<opset1::Constant>(data_prec, scale_shape, 4.f);
+            const auto shift = std::make_shared<opset1::Constant>(data_prec, shift_shape, 12.f);
+
+            const auto fake_convert = default_shift
+                                          ? std::make_shared<opset13::FakeConvert>(data, scale, dst_prec)
+                                          : std::make_shared<opset13::FakeConvert>(data, scale, shift, dst_prec);
+            model = std::make_shared<ov::Model>(OutputVector{fake_convert}, ParameterVector{data});
         }
 
-        model_ref = std::make_shared<ov::Model>(OutputVector{result}, params);
+        {
+            const auto data = std::make_shared<opset1::Parameter>(data_prec, PartialShape(data_shape));
+            const auto input_scale = std::make_shared<opset1::Constant>(data_prec, scale_shape, 4.f);
+            const auto input_shift = std::make_shared<opset1::Constant>(data_prec, shift_shape, 12.f);
+
+            const auto lower_bound = dst_prec == ov::element::f8e4m3
+                                         ? static_cast<float>(std::numeric_limits<ov::float8_e4m3>::lowest())
+                                         : static_cast<float>(std::numeric_limits<ov::float8_e5m2>::lowest());
+            const auto upper_bound = dst_prec == ov::element::f8e4m3
+                                         ? static_cast<float>(std::numeric_limits<ov::float8_e4m3>::max())
+                                         : static_cast<float>(std::numeric_limits<ov::float8_e5m2>::max());
+
+            std::shared_ptr<Node> result;
+            const auto scale = std::make_shared<ov::op::v1::Multiply>(data, input_scale);
+            if (default_shift) {
+                const auto clamp = std::make_shared<ov::op::v0::Clamp>(scale, lower_bound, upper_bound);
+                const auto downconvert = std::make_shared<ov::op::v0::Convert>(clamp, dst_prec);
+                result = std::make_shared<ov::op::v0::Convert>(downconvert, data_prec);
```

Comment:
Do I miss something, but why is upconvert for default_shift case not needed anymore?